draft-ietf-scitt-architecture.md

```diff
@@ -639,31 +639,24 @@ Although this weakens their guarantees against key revocation, or against a corr
 
 This section details the interoperability requirements for implementers of Signed Statements issuance and validation libraries, and of Transparency Services.
 
-##  Envelope and Signed Statement Format
+##  Issuing Signed Statements
 
-The formats of Signed Statements and Receipts are based on CBOR Object Signing and Encryption (COSE {{-COSE}}).
-The choice of CBOR {{-CBOR}} is a trade-off between safety (in particular, non-malleability: each Signed Statement has a unique serialization), ease of processing and availability of implementations.
-
-At a high-level that is the context of this architecture, a Signed Statement is a COSE single-signed object (i.e., a `COSE_Sign1`) that contains the correct set of protected headers.
-Although Issuers and relying parties may attach unprotected headers to Signed Statements, Transparency Services and Verifiers MUST NOT rely on the presence or value of additional unprotected headers in Signed Statements during Registration and validation.
+Signed Statements are CBOR encoded {{-CBOR}} and protected by CBOR Object Signing and Encryption (COSE {{-COSE}}). Additionally, it contains a minimum amount of headers and Statements as its payload.
```

Comment:
Line 644 does not look right to me? Assuming that you want atleast one header and a statement as payload I will re-write the statement,

Reply:
Accepted your wording suggestion